### PR TITLE
Update gleam to 0.3

### LINKED ISSUE #21: update gleam to 0.3
For servo/webrender#771, we need to update gleam to 0.3.

### PR DESCRIPTION
**Pull request by sotaroikeda**
This fixes #21.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/cgl-rs/22)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "cgl"
 license = "MIT / Apache-2.0"
-version = "0.1.5"
+version = "0.1.6"
```
Comment (jdm) [inline Cargo.toml]: This should be 0.2, since we use types from gleam in the publically-exported API in cgl-rs.

**Comment by sotaroikeda** [comment]
Updated the version to "0.2.0".

**Comment by sotaroikeda** [comment]
Thanks! Can it be updated on cargo?

**Comment by mbrubeck** [comment]
Published: https://crates.io/crates/cgl/0.2.0

**Pull request merged**